dist/stat_data.py

```diff
@@ -327,6 +332,22 @@ def __init__(self, name, desc, flags=''):
     CapacityStat('fsync_all_fh_total', 'background fsync file handles considered'),
     CapacityStat('fsync_all_time', 'background fsync time (msecs)', 'no_clear,no_scale'),
 
+    ##########################################
+    # Chunk cache statistics
+    ##########################################
+    ChunkCacheStat('chunk_cache_bytes', 'total bytes'),
+    ChunkCacheStat('chunk_cache_chunks', 'total chunks'),
```

Comment:
Is this the number of chunks currently in the cache? If so, I suggest `chunk_cache_chunks_inuse` and a description of `chunks currently held in the chunk cache`